### PR TITLE
Notifications module

### PR DESCRIPTION
**Pull request by MarcinMiler**

### CODE REVIEW

File: web/src/Routes/index.tsx [hunk 1]
```diff
@@ -2,12 +2,17 @@ import * as React from 'react'
...
 
 import { history } from 'Config/history'
+import { NotificationContainer } from 'Modules/Notification/Containers/Notifications'
+import { NotificationsWrapper } from 'Components/NotificationsWrapper'
 import { routes } from './routes'
 import { Navbar } from '../Components'
 
 export const Routes = () => (
     <Router history={history}>
         {/* <Navbar /> */}
```
Comment (mateusz-bielak) [inline web/src/Routes/index.tsx]: Maybe remove that commented component?

File: web/src/Routes/index.tsx [hunk 2]
```diff
@@ -2,12 +2,17 @@ import * as React from 'react'
 import { Router, Switch, Route } from 'react-router-dom'
 
 import { history } from 'Config/history'
+import { NotificationContainer } from 'Modules/Notification/Containers/Notifications'
+import { NotificationsWrapper } from 'Components/NotificationsWrapper'
 import { routes } from './routes'
 import { Navbar } from '../Components'
```
Comment (mateusz-bielak) [inline web/src/Routes/index.tsx]: Are you using it anywhere? Didn't linter warn you about unused variable?

File: web/src/Modules/Notification/factory.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const registerInfo = {
+    title: 'Successfully registered',
+    description: 'Now you can login'
+}
+export const registerNotificationSucceed = () =>
+    createNotification({
+        ...registerInfo,
+        type: NotificationTypes.SUCCEED
+    })
```
Comment (mateusz-bielak) [inline web/src/Modules/Notification/factory.ts]: This is a trifle, but you're sometimes using empty lines between const's here and sometimes not. Is it intentionally?

File: web/src/Routes/index.tsx (hunk 2 above)
Comment (MarcinMiler) [reply web/src/Routes/index.tsx]: I use it below ⏬

File: web/src/Routes/index.tsx (hunk 1 above)
Comment (MarcinMiler) [reply web/src/Routes/index.tsx]: I need to make this navbar render on specific routes, so for now i just commented it

Comment (mateusz-bielak) [reply web/src/Routes/index.tsx]: Where? Only in a commented line 😄

File: web/src/Routes/index.tsx (hunk 2 above)
Comment (mateusz-bielak) [reply web/src/Routes/index.tsx]: Where? Only in a commented line 😄

Comment (MarcinMiler) [reply web/src/Routes/index.tsx]: I thought it was about Notification Components 🤪 Yea it's unused

File: web/src/Modules/Notification/factory.ts (hunk 3 above)
Comment (MarcinMiler) [reply web/src/Modules/Notification/factory.ts]: No, it should be empty line

**Pull request merged**